data/hai/hai reveal 1 intro.txt

```diff
@@ -792,24 +796,24 @@ mission "Hai Leaks Response 4"
 	on visit
 		dialog `You have returned, but the fate of the pirates and the captured Hai is still in doubt. Ensure that the Black Lich is defeated and that the fate of the captured Hai, whether deceased or safely with you, is certainly known.`
 	to complete
-		or
-			has "00 Hai Leaks Response 4B: declined"
-			has "Hai Leaks Response 4A: failed"
+		not "Hai Leaks Response 4A: active"
 	on complete
 		payment 1000000
 		event "waypoint pirates vanquished"
 		event "hai ladybug available"
 		event "humans adopt ladybugs" 20 30
 		log `Defeated the "Black Lich". They had managed to kidnap a number of Hai straight off the planet with a combination of trickery and speed. Rewarded well.`
 		conversation
-			`When you return, the Hai Captain thanks you profusely. "The Allhome government has authorized a reward of <payment> for helping to route the pirate problem," he explains while handing you the credits.`
 			branch "saved geocoris"
-				has "00 Hai Leaks Response 4B: declined"
+				has "ship model (all): Geodesic Geocoris"
+			`Shortly after you return, the Hai quartermaster thanks you profusely. "The Allhome government has authorized a reward of <payment> for helping to route the pirate problem," he explains while handing you the credits.`
 			`	He sighs deeply. "It is a shame though that the kidnapped Hai could not be saved, but the other Hai captains tell me it was a mess out there."`
 			`	In an oddly human gesture that feels very deliberate, he places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
 				goto end
 			label "saved geocoris"
-			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy.`
+			`You have defeated the Black Lich and also successfully reclaimed the Geocoris, and in doing so rescued the kidnapped Hai! On your way back to <planet> the crew you sent across to fly it home reported that something is different about it... It appears that it has been modified with systems of some kind, which are not of Hai design. It also appears that perhaps not everything about those systems is working wholly as intended.`
```

Comment:
I wonder if this might be better placed in a small "on enter" conversation when you fly into Allhome's system with the Geodesic Geocoris?

It feels out of place for it to be shown when you land, with dialogue coming up next.

Reply:
It can't be made to appear there *only* when you've actually got it though, if we could have conditional `on enter`'s then it would be there

Reply:
Also this is where it always triggered/was meant to trigger - it just used to be separate and would trigger independently